client/mass/about.ts

```diff
@@ -91,14 +91,14 @@ export class MassAbout {
 					.html(description)
 			else new Error('Missing cohort description')
 		}
-
 		if (opts.selectCohort?.prompt) {
 			this.dom.cohortPrompt = this.subheader
 				.append('div')
 				.style('margin-left', '10px')
 				.style('padding-top', '30px')
 				.style('padding-bottom', '10px')
 				.style('font-weight', 'bold')
+				.style('font-size', '1.2em')
```

Comment:
this causes radio buttons look noticeably small compared to text
try .style.zoom="120%" on the div element wrapping these

Reply:
When I scale, the way to zoom is to use the scale property, it looked like this, I wanted to keep the radio buttons aligned with the text. I will see how.
<img width="1554" alt="Screenshot 2024-12-17 at 1 03 54 PM" src="https://github.com/user-attachments/assets/6e123960-9c6e-4ff0-b3a5-c7f0dee3abc3" />